src/services/Firebase/Converters/CommunityConverter.tsx

```diff
@@ -1,21 +1,17 @@
 import { DocumentData, QueryDocumentSnapshot } from 'firebase/firestore';
-import { IcommunityData } from '../../../types/types';
+import { Icategory } from 'types/types';
 
 export const communityConverter = {
-  toFirestore: (community: IcommunityData): DocumentData => {
```

Comment:
See my comment about not relying on categories to identify a community as active.

Reply:
this works better and it is more simple.